packages/elements/src/pagination/index.ts

```diff
@@ -426,23 +456,26 @@ export class Pagination extends BasicElement {
    * @return {TemplateResult} Render template
    */
   protected render (): TemplateResult {
+    let inputValue;
+    if (this.inputEditing) {
+      inputValue = this.value;
+    }
+    else {
+      inputValue = this.infinitePaginate ? this.t('PAGE', { page: this.value }) : this.t('PAGE_OF', { page: this.value, pageTotal: this.totalPage });
+    }
```

Comment:
Move this to a separate getter called `inputText`

Reply:
Updated